apps/prairielearn/src/lib/config.ts

```diff
@@ -477,7 +477,9 @@ const ConfigSchema = z.object({
    * create a course if the course request meets certain criteria.
    */
   courseRequestAutoApprovalEnabled: z.boolean().default(false),
-  devMode: z.boolean().default((process.env.NODE_ENV ?? 'development') === 'development'),
+  devMode: z
+    .boolean()
+    .default(['development', 'test'].includes(process.env.NODE_ENV ?? 'development')),
```

Comment:
This was very important, as `vitest` will set `NODE_ENV` to `test`.

Reply:
This is a nit, but can we pull out this logic to a const at the top of the file?

Reply:
Done in 683f24e. This was actually a good opportunity to do some more general cleanup. Previously we had different, inconsistent ways of doing this check in this file. I standardized on `process.env.NODE_ENV !== 'production'` as meaning dev mode.